CONTRIBUTING.md

```diff
@@ -202,3 +202,27 @@ Snapshot management requires VM t be powered off.
 
 ### QEMU on MacOS
 
+## FAQ
+
+### The `error sed: 1: "assets/Manifest": command a expects \ followed by text` or `command md5sum not found` has been faced
+when I try to run the `install.sh` script.
+
+The reason for that is because of the commands `sed -i` used on the script. Note that kURL
+only supports linux environments and then, you are probably trying to run it from a mac os. You can
+workarround by installing `gnu-sed` and `md5sum` with (`brew install gnu-sed`, `brew install md5sum`).
+However, to test the project you must have a `linux/amd64` platform to perform its operations.
```

Comment:
We try to only break up markdown sentences per line for simpler diffs.

```suggestion
The reason for that is because of the commands `sed -i` used on the script.
Note that kURL only supports linux environments and then, you are probably trying to run it from a mac os.
You can workaround by installing `gnu-sed` and `md5sum` with (`brew install gnu-sed`, `brew install md5sum`).
However, to test the project you must have a `linux/amd64` platform to perform its operations.
```